crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -203,6 +203,55 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
+    /// Returns an [`EntityCommands`] for the requested [`Entity`].
+    ///
+    /// In case of a nonexisting entity, a [`None`] is
+    /// returned instead.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// use bevy_ecs::prelude::*;
+    ///
+    /// #[derive(Component)]
+    /// struct Label(&'static str);
+    /// #[derive(Component)]
+    /// struct Strength(u32);
+    /// #[derive(Component)]
+    /// struct Agility(u32);
+
+    /// fn example_system(mut commands: Commands) {
+    ///     // Create a new entity with the needed component bundle
+    ///     let entity = commands
+    ///     .spawn_bundle((Strength(1), Agility(2)))
+    ///     .id();
+    ///     // Check (most likely in another System) if the entity still/already exists
```

Comment:
This comment is unclear / seems wrong; based on the code, it seems like this will be evaluated as part of the command processing automatically. That is in fact much more useful; it allows users to write commands that fail safely if the chosen entity does not exist.